### ISSUE TITLE
[Feature Request] Record without sound.

### ISSUE THREAD

**Issue by gaveitatry2** [koushick123/FlipCam#11]
@koushick123 - A feature you could add is to make videos NOT record sound. I don't want the video viewers hearing my girlfriend talk in the video recording, and no sound means smaller size videos. By the way, thank you for your fantastic app. It's 99% perfect and the best camera app on Google Play.

**Comment by koushick123** [comment]
Pushed a version 1.6 to record without sound. Available on Play Store.

**Comment by gaveitatry2** [comment]
@koushick123 - Thank you for adding this feature! I checked just now and my app had finally updated and the record without sound feature seems to be working. It's too bad that I stopped recording videos. But if I ever record videos again, I'll have your awesome app on my phone to record with.

My only other suggestion for this app is a custom location (on SD Card Drive). I can't save to Phone Memory because there isn't much space. So I have to save to SD Card. But the predetermined place for SD Card is (you know) and when I go looking for the videos I recorded, I have to dig around in subfolders. But if I could change the SD Card location to the main parent SD Card drive folder instead of /storage/.../Android/media/com.flipcam... then I wouldn't have to dig around in subfolders and the videos would show up in my apps right away. Just an idea. I think that would be an awesome feature as well. But thank you for adding this record without sound feature.
